example/controller-registration.yaml

```diff
@@ -6,11 +6,13 @@ metadata:
 spec:
   resources:
   - kind: OperatingSystemConfig
-    type: metal
+    type: ubuntu
```

Comment:
Unfortunately, we overlap with the Gardener os-controller for ubuntu, which also wants to watch "ubuntu" types. This leads to the controllers going wild. We should rename this to `ubuntu-metal` or `ubuntu-ignition`, same for the debian image.

Reply:
Added a comment to upstream:
https://github.com/gardener/gardener-extensions/pull/580#issuecomment-578786683